how-to/wireguard-vpn/site-to-site.md

```diff
@@ -3,26 +3,37 @@
 
 Another usual VPN configuration where one could deploy WireGuard is to connect two distinct networks over the internet. Here is a simplified diagram:
 
+The following code-block will be rendered as a Mermaid diagram:
+
+```mermaid
```

Comment:
```suggestion
```{mermaid}
```

Reply:
I've tested on my own local branch and putting the curly brackets around mermaid should "encourage" Sphinx to recognize this as a mermaid diagram :) if you can apply this change, when the preview is rebuilt it should show the diagram properly